src/deterministic_vi/elbo.jl

```diff
@@ -235,11 +235,12 @@ function accum_star_pos!{NumType <: Number}(
                     x::Vector{Float64},
                     wcs_jacobian::Array{Float64, 2},
                     calculate_derivs::Bool)
+
     eval_bvn_pdf!(elbo_vars.bvn_derivs, bmc, x)
 
-    # TODO: Also make a version that doesn't calculate any derivatives
-    # if the object isn't in active_sources.
-    get_bvn_derivs!(elbo_vars.bvn_derivs, bmc, true, false)
+    if elbo_vars.calculate_derivs && calculate_derivs
```

Comment:
Could we change the name of the local `calculate_derivs` to something like `is_active_source`?